examples/example.c

```diff
@@ -556,12 +556,8 @@ main(int argc, char **argv)
         sentry_capture_event(event);
     }
     if (has_arg(argc, argv, "capture-user-feedback")) {
-        sentry_value_t event = sentry_value_new_message_event(
-            SENTRY_LEVEL_INFO, "my-logger", "Hello user feedback!");
-        sentry_uuid_t event_id = sentry_capture_event(event);
-
         sentry_value_t user_feedback = sentry_value_new_user_feedback(
-            &event_id, "some-name", "some-email", "some-comment");
+            NULL, "some-name", "some-email", "some-message");
```

Comment:
If we keep the old implementation and do not only replace it with the new one (which it seems like we could, without any effects to the user, since the new one takes an optional event), then we should keep the example and integration test for the old.